c-api/compat-5.3.h

```diff
@@ -88,6 +86,13 @@ extern "C" {
 #  define LUA_OPLE 2
 #endif
 
+// LuaJIT/Lua 5.1 does not have the updated 
```

Comment:
Use `/*` style comments for consistency and compatibility.

Reply:
I went through and fixed all the comments.